src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```
@@ -219,14 +220,20 @@ public boolean isEmpty() {
      * @return A list of all variant headers for features.
      */
     public List<VCFHeader> getAllVariantHeaders() {
-        List<VCFHeader> headers = new ArrayList<>();
-        for (Map.Entry<FeatureInput<? extends Feature>, FeatureDataSource<? extends Feature>> entry : featureSources.entrySet()) {
-            final Object header = entry.getValue().getHeader();
-            if ( header != null && header instanceof VCFHeader ) {
-                headers.add((VCFHeader)header);
-            }
-        }
-        return headers;
+        return featureSources.values().stream()
+                .map(feature -> feature.getHeader())
+                .filter(header -> header instanceof VCFHeader)
+                .map(header -> (VCFHeader)header).collect(Collectors.toList());
+    }
+
+    /**
+     * Returns the list of sequence dictionaries retrieved from the headers.
```

Also mention that this will silently return an empty `List` if the variant inputs happen not to have sequence dictionaries (since they are optional in the VCF format).